### PR TITLE
Rename smokey-nagios to smokey-loop

### PR DESCRIPTION
**Pull request by alexmuller**
This confuses everyone who comes across it (including me). The fact that smokey runs in a loop is hardly anything to do with the fact it's consumed by Icinga (not Nagios).

### CODE REVIEW

**Comment by surminus** [comment]
I also love it how [this](https://github.com/alphagov/govuk-puppet/blob/fd51ac0c86f9c481650130d8b21c155d041e5b8a/modules/monitoring/files/etc/init/smokey-nagios.conf#L9) is run as a service in a loop rather than in a cron job

**Comment by alexmuller** [comment]
Yeah, it's a bit different. I think we do it like that because we want it running as frequently as possible. But it's been a long time.

**Comment by surminus** [comment]
Yeah, I just meant the name of it is "cron" even though it's not used anywhere as a cronjob.

Anyway, :+1: fftm

**Comment by alexmuller** [comment]
Oh yeah, I totally missed that. That's silly. Hold tight, let's fix it.

**Comment by alexmuller** [comment]
https://github.com/alphagov/smokey/pull/178

File: modules/monitoring/manifests/checks/smokey.pp [hunk 1]
```diff
@@ -14,16 +14,26 @@
...
+  service { 'smokey-nagios':
+    ensure => stopped,
+  } ->
   file { '/etc/init/smokey-nagios.conf':
+    ensure => absent,
+  }
+
+  $service_file = '/etc/init/smokey-loop.conf'
+
+  # TODO: Should this really run as root?
```
Comment (deanwilson) [inline modules/monitoring/manifests/checks/smokey.pp]: Probably not. Add an investigation to the ideas backlog?

**Comment by deanwilson** [comment]
Merge away.

**Pull request merged**